contracts/staking/ValidatorManager.sol

```diff
@@ -93,7 +95,17 @@ abstract contract ValidatorManager is
         ValidatorManagerStorage storage $ = _getValidatorManagerStorage();
         $._pChainBlockchainID = settings.pChainBlockchainID;
         $._subnetID = settings.subnetID;
-        $._maximumHourlyChurn = settings.maximumHourlyChurn;
+
+        require(
+            settings.maximumChurnPercentage <= 20,
+            "ValidatorManager: maximum churn percentage too high"
+        );
```

Comment:
We could also enforce maximum churn percentage > 0, but setting it to 0 to disable is useful at least for tests.

Reply:
0 seems to be an invalid value to me

Reply:
Disabling churn tracker should only be useful in tests though right?

Reply:
Yeah, I guess with Stephen's comments about consensus failures, we should probably require it to be enabled. 

I think now that we're setting an initial validator set, tests should be easy enough to get to work around the churn tracker limits.